scanpy/external/_tools/_palantir.py

```diff
@@ -133,7 +133,7 @@ def palantir(adata: AnnData):
 
     """
 
-    logg.info('Palantir diffusion maps', r=True)
+    logg.info('Palantir diffusion maps')
```

Comment:
If you don't reset (or start) a timer, there is no way at which point the timer should know when to start timing. So, I fear, there needs to be an additional parameter here.

Reply:
I checked and it seems like automatically resetting at every `info` call works, as there seem to be no `info` loggings in the middle of some long running process.